### PR TITLE
Overture version upgrade

### PR DESCRIPTION
**Pull request by donbex**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import com.anaplan.buildtools.jenkins_pipelines.ContainerTemplates
+import com.anaplan.buildtools.jenkins_pipelines.DefaultConfig
+
+@Library('Anaplan_Pipeline')
+
+def BUILD_LABEL = "azuki.${UUID.randomUUID().toString()}"
```
Comment (sifraser) [inline Jenkinsfile]: Not azuki

**Review by sifraser — CHANGES_REQUESTED** [review]

File: build.gradle [hunk 2]
```diff
@@ -38,9 +37,25 @@ dependencies {
 
     implementation group: 'org.overturetool.core', name: 'interpreter', version: overtureVersion
 
-    testImplementation group: 'junit', name: 'junit', version: '4.12'
-    testImplementation group: 'com.nhaarman', name: 'mockito-kotlin', version: '1.5.0'
+    testImplementation group: 'junit', name: 'junit', version: '4.13.2'
```
Comment (sifraser) [inline build.gradle]: we should change this to use `kotlin-test` while we are here

**Comment by sifraser** [comment]
Should probably also add `.editorconfig`

File: Jenkinsfile (hunk 1 above)
Comment (donbex) [reply Jenkinsfile]: 🤦

**Comment by donbex** [comment]
> Should probably also add `.editorconfig`

I was planning to do all the re-formatting in a separate PR, but I can do it here if you prefer.

File: src/test/kotlin/com/anaplan/engineering/vdmprettyprinter/VdmSlReparseTest.kt [hunk 3]
```diff
@@ -21,9 +21,9 @@
  */
 package com.anaplan.engineering.vdmprettyprinter
 
-import junit.framework.TestCase.assertEquals
+import kotlin.test.assertEquals
 import org.junit.Assume
-import org.junit.Test
+import kotlin.test.Test
 import org.junit.runner.RunWith
 import org.junit.runners.Parameterized
```
Comment (donbex) [inline src/test/kotlin/com/anaplan/engineering/vdmprettyprinter/VdmSlReparseTest.kt]: Looks like `kotlin-test` only provides assumption checks.

**Review by sifraser — APPROVED** [review]

**Pull request merged**